app/src/Staging/java/com/simpragma/recipe/recipeapplication1/Constants1.java

```diff
@@ -0,0 +1,11 @@
+package com.simpragma.recipe.recipeapplication1;
+
+/**
+ * Created by shubhank on 2/5/18.
+ */
+
+class Constants1 {
```

Comment:
Why code is repeated? Why do we have folder structure like Staging?

Reply:
Because i am checking staging and production type

Reply:
i will check it .If possible i will change code